charts/node/values.yaml

```diff
@@ -77,7 +77,7 @@ node:
   collator:
     isParachain: false
     relayChain: polkadot
-  #database: rocksdb
+  database: rocksdb
```

Comment:
A linter throws a warning because you're trying to use an undefined variable in `_helpers.tpl`

```
[ERROR] templates/: template: node/templates/statefulset.yaml:5:20: executing "node/templates/statefulset.yaml" at <include "node.databasePath" .>: error calling include: template: node/templates/_helpers.tpl:91:7: executing "node.databasePath" at <eq .Values.node.database "paritydb">: error calling eq: incompatible types for comparison
```